compiler/rustc_codegen_llvm/src/gotoc/mod.rs

```diff
@@ -36,6 +39,60 @@ pub mod stubs;
 mod typ;
 mod utils;
 
+// Use a thread-local global variable to track the current codegen item for debugging.
+// If RMC panics during codegen, we can grab this item to include the problematic
+// codegen item in the panic trace.
+thread_local!(static CURRENT_CODEGEN_ITEM: RefCell<Option<String>> = RefCell::new(None));
+
+// Updates the tracking global variable for panic debugging.
+fn set_debug_item(s: String) {
```

Comment:
Is there a more descriptive name to use here?

Reply:
Changed, thanks!